apps/docs/deploying-codemods/cli.mdx

```diff
@@ -33,7 +33,7 @@ npm i -g codemod
 The <Tooltip tip='Available aliases: `ls` and `search`'>`list`</Tooltip> command can be used to list the codemods available in the Codemod Registry and optionally search for a specific codemod using its name or tags.
 
 ```bash
-codemod list [optional search query]
+codemod list [optional-search-query]
```

Comment:
why dont we use codemod search by default? @r4zendev @mohab-sameh

Reply:
@alexbit-codemod both commands work the same. It's just that `list` makes more sense when used alone like:
```bash
codemod list
```

instead of:
```bash
codemod search
```

So we used this as the default command.